<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>183</width>
      <height>128</height>
     </rect>
    </property>
    <property name="title">
     <string>Pattern Setting</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_R">
         <property name="text">
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_R">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_R">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_G">
         <property name="text">
          <string>G:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_G">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_G">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_B">
         <property name="text">
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_B">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_B">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>217</width>
      <height>128</height>
     </rect>
    </property>
    <property name="title">
     <string>Ratio Setting</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ratioR:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_ratioR">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_ratioR">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="value">
          <number>4095</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ratioG:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_ratioG">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_ratioG">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="value">
          <number>4095</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ratioB:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_ratioB">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_ratioB">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="value">
          <number>4095</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_screenWidth">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1920</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_screenHeight">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1080</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fullscreen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FULLSCREEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fullscreen_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出全屏</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>43</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
